examples/sudt-transfer.rs

```diff
@@ -71,8 +71,12 @@ enum Commands {
 struct Config {
     #[serde(flatten)]
     sdk_config: SdkConfig,
-    /// End user sighash(secp256k1) private key in hex without 0x prefix.
-    private_key: String,
+    /// Specify end user sighash(secp256k1) private key in hex without 0x
+    /// prefix, or read the key from a binary file, or pass the key (still in
+    /// hex without 0x prefix) via the SUDT_TRANSFER_PRIVATE_KEY environment
+    /// variable.
+    private_key: Option<String>,
+    private_key_file: Option<String>,
```

Comment:
I need here to be env var, letting me write private-key in GitHub secret manager

Reply:
The private key is read from the `SUDT_TRANSFER_PRIVATE_KEY` env var when both options are none.